src/Abuse/Adapters/TimeLimit/Database.php

```diff
@@ -246,17 +249,18 @@ public function getLogs(?int $offset = null, ?int $limit = 25): array
     /**
      * Delete logs older than $timestamp seconds
      *
-     * @param  string  $datetime
+     * @param  int  $timestamp
      * @return bool
      *
      * @throws AuthorizationException|\Exception
      */
-    public function cleanup(string $datetime): bool
+    public function cleanup(int $timestamp): bool
     {
-        Authorization::skip(function () use ($datetime) {
+        $timestamp = $this->toDateTime($timestamp);
+        Authorization::skip(function () use ($timestamp) {
             do {
                 $documents = $this->db->find(self::COLLECTION, [
-                    Query::lessThan('time', $datetime),
+                    Query::lessThan('time', $timestamp),
```

Comment:
Dont we need to change attribute type too?

Reply:
in the database adapter, I [convert the timestamps](https://github.com/utopia-php/abuse/pull/91/files#diff-5251a91329b67cf39cdc8d4eea5d4edeb4b326287ff6df7ed167492cd8239feaR139) to the required format